LDAP-Auth/LDAP-Auth.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
build.yaml is the important file for CI, please include the changes to that file from #59 

```yaml
version: "10.0.0.0"
targetAbi: "10.7.0.0"
framework: "net5.0"
```